src/layout/Footer/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+    background-color: #22272e;
+    color: #adbac7;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji";
```

Comment:
Nice use of the actual github font! How did you implement this? We tried but couldn't figure it out